Cargo.toml

```
@@ -10,23 +10,23 @@ repository = "https://github.com/awslabs/coldsnap"
 keywords = ["AWS", "Amazon", "EBS", "snapshot"]
 
 [features]
-default = ["rusoto-native-tls"]
-rusoto-native-tls = ["rusoto_core/native-tls", "rusoto_ebs/native-tls", "rusoto_ec2/native-tls"]
-rusoto-rustls = ["rusoto_core/rustls", "rusoto_ebs/rustls", "rusoto_ec2/rustls"]
+default = ["aws-sdk-rust-native-tls"]
+aws-sdk-rust-native-tls = ["aws-config/native-tls", "aws-sdk-ebs/native-tls", "aws-sdk-ec2/native-tls"]
+aws-sdk-rust-rustls = ["aws-config/rustls", "aws-sdk-ebs/rustls", "aws-sdk-ec2/rustls"]
 
 [dependencies]
 argh = "0.1.7"
-tokio = { version = "1.8", features = ["fs", "io-util", "time"] }
+aws-config = "0.11.0"
+aws-sdk-ebs = "0.11.0"
+aws-sdk-ec2 = "0.11.0"
+aws-smithy-http = "0.41.0"
+aws-types = "0.11.0"
+tokio = { version = "1", features = ["full"] }
```

Do we need all the features of `tokio`, or can we use our usual set?
```suggestion
tokio = { version = "~1.14", default-features = false, features = ["macros", "rt-multi-thread"] }  # LTS
```

---

aws sdk rust specifies "full" https://github.com/awslabs/aws-sdk-rust. I'm unsure of the consequences of not using full, so I followed the docs

---

> Do we need all the features of tokio

In your suggestion you regressed us back to locking the version! I recommend we stick with `1` unless there is a reason to force a version minimum on downstream lib users.

---

I agree with @webern that we don't want to pin the tokio version.

That example in the aws-sdk-rust readme doesn't really persuade me; they might've just picked "full" because it was convenient - easy for developers to add, unlikely to change over time.

It looks like they're being more specific about tokio features in their own crates.

---

Putting that all together, I think we want this:
```
tokio = { version = "1", features = ["fs", "io-util", "time"] }
```

Like it was before, but using `1` instead of `1.8`.

---

Side note: if the SDK chose full features, then that's what we're getting anyway. But I agree with the best practice of specifying our minimum feature set. Pretty sure we will need `rt-multi-threaded` and `macros` as well. The silly thing is that we wouldn't know whether we needed them or not because, if the SDK includes `full` then those features are there.

Anyway if this message is confusing, ignore it.